### PR TITLE
Document divergence from module spec

### PR DESCRIPTION
**Pull request by erights**
New insight in the second paragraph.

### CODE REVIEW

File: doc/divergence.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+   * The emulated behavior is that this instead reads an `undefined`.
+
+A only has read access to V, so there is no issue of A assigning to V. Aside from import cycles, this divergence should not be observable. Since the divergence turns an error condition into a non-error, it is unlikely to break the functionality of any programs other than test cases. Since the non-error provides no more power than the expected error, it is unlikely to introduce any exploitable vulnerabilities.
+
+The constraints and goals that together cause this divergence:
+
+   * An exporting module can export a live binding, i.e., a non-const binding that it subsequently assigns.
+   * The source code of an importing module does not distinguish an imported live binding from an imported stable binding.
+   * We translate each module separately, without any inter-module analysis. Thus, our translation must assume that any imported variable might be a live binding.
+   * The translation translates only import declarations, export declarations, and eventually dynamic import expressions and `import.meta` expressions. All other source text must remain exactly as it originally appeared.
```
Comment (michaelfig) [inline doc/divergence.md:21]: Dynamic import expressions and `import.meta` expressions are already translated.

**Review by jfparadis — COMMENTED** [review]
Perfect!

**Review by michaelfig — APPROVED** [review]
LGTM!

File: doc/divergence.md (hunk 1 above)
Comment (erights) [reply doc/divergence.md:21]: Done

**Pull request merged**